client/src/components/TeamManagement/index.tsx

```diff
@@ -0,0 +1,103 @@
+import { inviteAdminUser, useAdminUsers } from '@enterprise-oss/osso';
+import { Button, Card, Form, Input, Modal, Select, Table } from 'antd';
+import React, { ReactElement, useState } from 'react';
+
+import styles from './index.module.css';
+
+export default function TeamManagement(): ReactElement {
+  const { data, loading } = useAdminUsers();
+  const { inviteUser } = inviteAdminUser();
+  const [modalOpen, setModalOpen] = useState(false);
+  const [form] = Form.useForm();
+
+  return (
+    <Card>
+      <div className={styles.cardTopRow}>
+        <h2>Team</h2>
+        <Button size="small" onClick={() => setModalOpen(true)}>
+          Invite Teammate
+        </Button>
+      </div>
+      <Table
+        loading={loading}
+        rowKey="id"
+        dataSource={data?.adminUsers}
+        className={styles.table}
+      >
+        <Table.Column title="Email" dataIndex="email" key="email" />
+        <Table.Column
+          title="Last login"
+          dataIndex="updatedAt"
```

Comment:
IDK how you're running the timestamp here, but is it possible to change to a more human readable format? I.e. "6 hours ago" or even `DD/MM/YY HH:MM`

Reply:
these are also faked rn to -12 hours. do you think this column is important to the table?

Reply:
not particularly, we can start to add data in as users request it (or during some research I can uncover what is important to include here)